src/exporter/models/exporterStore.ts

```diff
@@ -86,9 +75,47 @@ export const exporterStore = types
         }
       }
     );
+    const getGeoPackages: () => Promise<void> = flow(
```

Comment:
rename to getExportTasks

Reply:
Done, but in other brunch unit tests will be effected (we need to minimize time to PR review)